modules/abstract-utxo/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bitgo/abstract-utxo",
-  "version": "2.1.2",
+  "version": "2.2.1",
```

Comment:
If we are going from version 2.1.2 and adding features (but no breaking changes), shouldn't the new version be 2.2.0?

Reply:
I'd have to look at all the commits. This PR has a couple stable releases in it, so 2.2.0 was probably farther back in the commit history